lib/transform/transformers.js

```diff
@@ -20,6 +20,16 @@ export function webhooks (webhook) {
   if (webhook.httpBasicUsername) {
     delete webhook.httpBasicUsername
   }
+
+  // Workaround for webhooks with secret headers
+  if (webhook.headers) {
+    webhook.headers = webhook.headers.reduce((accumulator, header) => {
```

Comment:
Any reason to not use `Array.prototype.filter` instead?

```
webhook.headers = webhook.headers.filter((header) => !header.secret)
```

Reply:
nope just slipped my mind :) will change